packages/libs/eda/src/lib/core/components/visualizations/implementations/LineplotVisualization.tsx

```diff
@@ -970,7 +970,7 @@ function LineplotViz(props: VisualizationProps<Options>) {
     }
     return Object.entries(dataElementConstraints[0])
       .filter((variable) => variable[1].isRequired)
-      .every((reqdVar) => !!(vizConfig as any)[reqdVar[0]]);
+      .every((reqdVar) => !!vizConfig[reqdVar[0] as keyof typeof vizConfig]);
```

Comment:
I think you can use `LineplotConfig` here
```suggestion
      .every((reqdVar) => !!vizConfig[reqdVar[0] as keyof LineplotConfig]);
```

Reply:
👍 complied